tests/comparison.test.js

```diff
@@ -0,0 +1,44 @@
+var describe = require('mocha').describe
+var it = require('mocha').it
+var expect = require('chai').expect
+var strictEqual = require('../comparison')
+
+describe('comparison', () => {
+    describe('strictEqual', () => {
+        it('returns true when given two parameters of the same type and same value', () => {
+            // setup test data
+            //call the function with the test data 
```

Comment:
Comments like this are great when we're working out how things should be laid out, but we should always delete them once we're done so they don't pollute repo history.